controller/test/testmanager.spec.ts

```diff
@@ -37,8 +37,10 @@ import type { File, FileJSON } from "formidable";
 import { Test as MochaTest } from "mocha";
 import { PpaasEncryptS3File } from "../pages/api/util/ppaasencrypts3file";
 import { TestSchedulerIntegration } from "./testscheduler.spec";
+import { VERSION_TAG_NAME } from "../pages/api/util/pewpew";
 import { expect } from "chai";
 import { latestPewPewVersion } from "../pages/api/util/clientutil";
+import { mockGetObjectTagging } from "./mock";
```

Comment:
You'll also need to pull in `mockS3` to call in the `before()` method and `resetMockS3` to call in the `after` method of the tests